djvm/src/main/kotlin/net/corda/djvm/SandboxConfiguration.kt

```diff
@@ -34,8 +34,8 @@ class SandboxConfiguration private constructor(
         val ALL_RULES: List<Rule> = unmodifiableList(listOf(
             AlwaysUseNonSynchronizedMethods,
             AlwaysUseStrictFloatingPointArithmetic,
-            DisallowDynamicInvocation,
```

Comment:
Is the policy regarding `invokedynamic` changing here? Is this implied by CORDA-3062 or a separate issue?

Reply:
Technically, there is no change of policy here. I discovered that the `DisallowDynamicInstruction` rule was never being invoked because _all_ `InstructionRule`s had accidentally been disabled. (`DisallowDynamicInstruction` was the _only_ such rule, but this PR is adding a new one.)

I have now re-implemented this `Rule` as an `Emitter` because we cannot refuse to load a class just for _having_ an `invokedynamic` instruction somewhere. However, I have also disabled the emitter because we _need_ "safe" lambda support! Corda is full of lambdas, for instance.

So the net effect is "Unapplied `Rule`" -> "Disabled `Emitter`", i.e. no change :grinning:.